.circleci/config.yml

```diff
@@ -296,13 +296,13 @@ jobs:
           name: Check Python dependencies for CVEs
           command: |
             fromtag=$(docker images |grep securedrop-test-xenial-py3 |head -n1 |awk '{print $2}')
-            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-xenial-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "sudo pip3 install -q --upgrade safety && make -C .. safety"
+            DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-xenial-py3:${fromtag:-latest}" securedrop/bin/dev-shell bash -c "pip3 install -U -q --upgrade safety && make -C .. safety"
```

Comment:
`-U` is redundant with `--upgrade` already in the task list. Seems like it's lack of sudo privileges that would have resolved  #5748. Not a blocker, since today is release day and CI runs take a long time, just pointing it out in case I'm missing something.